tool/gravity/cli/operation.go

```diff
@@ -265,6 +318,16 @@ func (r *backendOperations) getOperationAndUpdateCache(backend storage.Backend,
 }
 
 func (r backendOperations) isActiveInstallOperation() bool {
+	// Bail out if there's an operation from a local backend and we failed to query
```

Comment:
TBH I'm not sure I follow this comment - could it be reworded somehow maybe? To explain what "r.clusterOperation" is, what "r.operations" are and why this condition means this is not an active install operation.

Reply:
I'll update the struct responsible for operation collection.